gravitee-am-model/src/main/java/io/gravitee/am/model/User.java

```diff
@@ -176,6 +177,18 @@ public class User implements IUser {
 
     private Boolean serviceAccount = Boolean.FALSE;
 
+    private String employeeNumber;
+
+    private String costCenter;
+
+    private String organization;
+
+    private String division;
+
+    private String department;
+
+    private Manager manager;
```

Comment:
question: do we have to keep those attributes at this level, should we create an EnterpriseAttribute bean to make it clear? 🤔